packages/core-data/src/test/selectors.js

```diff
@@ -124,6 +124,7 @@ describe( '__experimentalGetDirtyEntityRecords', () => {
 						kind: 'someKind',
 						name: 'someName',
 						transientEdits: { someTransientEditProperty: true },
+						label: 'someLabel',
```

Comment:
@epiqueras -  Do you think its ok to add the `label` attribute to this `__experimentalGetDirtyEntityRecords` ?

It allows us to not have to useSelect for a redundant `getEntity` call here:
https://github.com/WordPress/gutenberg/blob/66792194130bbccad8fdae7ee36b938726ed3be9/packages/editor/src/components/entities-saved-states/index.js#L105-L109

I thought it would also help with generating the phrase but that did not end up being the case.